CaseStudy1/VehicleHelper

```diff
@@ -0,0 +1,15 @@
+// Class to create a vehicle type according to the user
+public class VehicleHelper {
+	// Static function to create car or bike type object
+	public static Vehicle createVehicle(String vehicleType, String make,
+			String model) {
+		if (vehicleType.equalsIgnoreCase("Car")) {
+			Car car = new Car(make, model);
+			return car;
+		} else {
```

Comment:
In the else use else if statement for bike and if vehicle type is not bike and car report error